src/main/java/se/bjurr/violations/lib/parsers/JacocoXmlParser.java

```diff
@@ -0,0 +1,138 @@
+package se.bjurr.violations.lib.parsers;
+
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+import org.w3c.dom.NodeList;
+import org.xml.sax.InputSource;
+import se.bjurr.violations.lib.ViolationsApi;
+import se.bjurr.violations.lib.ViolationsLogger;
+import se.bjurr.violations.lib.model.SEVERITY;
+import se.bjurr.violations.lib.model.Violation;
+import se.bjurr.violations.lib.reports.Parser;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import java.io.File;
+import java.io.StringReader;
+import java.util.HashSet;
+import java.util.Optional;
+import java.util.Set;
+import java.util.TreeSet;
+
+import static se.bjurr.violations.lib.model.Violation.violationBuilder;
+import static se.bjurr.violations.lib.reports.Parser.JACOCO;
+
+public class JacocoXmlParser implements ViolationsParser {
+
+    private static final int MIN_LINE_COUNT = 4;
+
+    private static final double MIN_COVERAGE = 0.7;
+
+    @Override
+    public Set<Violation> parseReportOutput(String reportContent, ViolationsLogger violationsLogger) throws Exception {
+        Set<Violation> violations = new HashSet<>();
+        DocumentBuilderFactory documentBuilderFactory = DocumentBuilderFactory.newInstance();

```

Comment:
The current xml-parsers use `XmlStreamReader`, I think I would prefer using that one everywhere.
https://github.com/tomasbjerre/violations-lib/blob/5fa2ec7c1852cd9749309b2ae5309ef9a7c30b9f/src/main/java/se/bjurr/violations/lib/parsers/FindbugsParser.java#L54

Reply:
I saw that XMLStreamReader was being used, for me the DOMParser was the easiest to publish something for feedback. Will change to the reader together with the tests.